test/ccall.jl

```diff
@@ -1263,3 +1263,16 @@ end
 (::CallableSingleton)(x, y) = x + y
 @test ccall(cfunction(CallableSingleton(), Int, Tuple{Int,Int}),
             Int, (Int, Int), 1, 2) === 3
+
+
+# 19805
+type callinfos_19805{FUNC_FT<:Function}
+  f :: FUNC_FT
```

Comment:
I don't see a reason to format user reported test cases. Feel free to format as you feel fit.